src/main/java/cms/gov/madie/measure/services/VersionService.java

```diff
@@ -124,6 +127,7 @@ public Measure createDraft(String id, String measureName, String username) {
     measureDraft.setCreatedAt(now);
     measureDraft.setLastModifiedAt(now);
     measureDraft.setCreatedBy(username);
+    measureDraft = setMeasureReviewMetaData(measureDraft);
```

Comment:
The Functional ACs call for the Approval Date and Last Review date to be **cleared** on draft, if present.

Reply:
@jkotanchik-SB Ok thanks will make the change.